scripts/gen-changelog.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import requests
+import os
+
+def get_all_releases(token):
+    url = f"https://api.github.com/repos/meta-llama/llama-stack/releases"
+    headers = {"Accept": "application/vnd.github.v3+json"}
+    
+    if token:
+        headers["Authorization"] = f"token {token}"
+    
+    response = requests.get(url, headers=headers)
+    
+    if response.status_code == 200:
+        return response.json()
+    else:
+        print(f"Error fetching releases: {response.status_code}, {response.text}")
```

Comment:
If it's an error, should we raise instead?

Reply:
Thanks. Updated